triggers/tests/src/test/scala/com/daml/trigger/test/TestMain.scala

```diff
@@ -335,7 +335,7 @@ case class AcsTests(dar: Dar[(PackageId, Package)], runner: TestRunner) {
       // 1 for the exercise in the trigger
       // 2 completions for the trigger
       NumMessages(6),
-      SuccessfulCompletions(3),
+      SuccessfulCompletions(2),
```

Comment:
These have decreased since we now only get completions for commands that we have emitted ourselves.